### DISCUSSION TITLE
Change time zone

### DISCUSSION THREAD

**Discussion by xde012** [Security-Onion-Solutions/securityonion#11350] [General] [answered]
hello,

I have read a lot that it is recommended to use the UTC time zone but in my case I want to change it, the thing is that for some reason SO changes it again automatically in a matter of minutes.

I am trying it in the following way:

sudo timedatectl set-timezone "Europe/Madrid"

Once done I get the expected result and the time zone changes to Madrid/Europe but in a matter of a few minutes I run "timedatectl" again and we return to the initial position.

Should this change be made in a file related to Salt?

Thank you so much.

**Comment by dougburks** [comment, marked as answer]
Security Onion sets its time zone to UTC intentionally. Please do not try to change it. Instead, simply allow the web interface to render timestamps in your local timezone.

From https://docs.securityonion.net/en/2.3/timezones.html#utc-and-time-zones:
> When you run Security Onion Setup, it sets the operating system timezone to UTC/GMT. Logging in UTC is considered a best practice across the cybersecurity industry because it makes it that much easier to correlate events across different systems, organizations, or time zones. Additionally, it avoids issues with time zones that have daylight savings time which would result in a one-hour time warp twice a year.
>
>Web interfaces like [Alerts](https://docs.securityonion.net/en/2.3/alerts.html#alerts), [Dashboards](https://docs.securityonion.net/en/2.3/dashboards.html#dashboards), [Hunt](https://docs.securityonion.net/en/2.3/hunt.html#hunt), and [Kibana](https://docs.securityonion.net/en/2.3/kibana.html#kibana) should try to detect the timezone of your web browser and then render those UTC timestamps in local time. [Alerts](https://docs.securityonion.net/en/2.3/alerts.html#alerts), [Dashboards](https://docs.securityonion.net/en/2.3/dashboards.html#dashboards), and [Hunt](https://docs.securityonion.net/en/2.3/hunt.html#hunt) allow you to manually set your timezone under Options.
